hedera-node/hedera-app/src/main/java/com/hedera/node/app/info/StateNetworkInfo.java

```diff
@@ -116,16 +112,11 @@ public boolean containsNode(final long nodeId) {
     @Override
     public void updateFrom(@NonNull final State state) {
         final var config = configProvider.getConfiguration();
-        if (config.getConfigData(TssConfig.class).keyCandidateRoster()) {
-            activeRoster = retrieveActiveOrGenesisRoster(state);
-        } else {
-            // When the feature flag is disabled, the rosters in RosterService state are not up-to-date
-            // FUTURE: Once TSS Roster is implemented in the future, this will be removed and use roster state
-            // instead of the address book
-            final var readablePlatformStateStore =
-                    new ReadablePlatformStateStore(state.getReadableStates(PlatformStateService.NAME));
-            activeRoster = buildRoster(requireNonNull(readablePlatformStateStore.getAddressBook()));
-        }
+
+        // RosterRetriever will fetch the roster from the RosterService state if it's populated.
+        // Otherwise, it falls back to reading the AddressBook from the PlatformStateService.
+        activeRoster = retrieveActiveOrGenesisRoster(state);
```

Comment:
This `activeRoster` field should really be `final` since rosters only change at upgrade boundaries; so this update is removed in https://github.com/hashgraph/hedera-services/pull/16843.

Reply:
I'll remove it if your PR makes it to `develop` first.